shared/src/vm/wasm/run.rs

```diff
@@ -488,6 +488,7 @@ mod tests {
     /// Test that when a transaction wasm goes over the stack-height limit, the
     /// execution is aborted.
     #[test]
+    #[cfg_attr(all(target_arch = "aarch64", target_os = "macos"), ignore)]
```

Comment:
```suggestion
    // Disabled since a fix for https://github.com/wasmerio/wasmer/issues/4072
    // reduced the available stack space on mac
    #[cfg_attr(all(target_arch = "aarch64", target_os = "macos"), ignore)]
```